python/modules/Stop0lBaselineProducer.py

```diff
@@ -136,21 +136,26 @@ def PassNjets(self, jets):
 
     def GetJetSortedIdx(self, jets):
         ptlist = []
+	etalist = []
         dphiMET = []
         for j in jets:
             if math.fabs(j.eta) > 4.7 or j.pt < 20:
                 pass
             else:
                 ptlist.append(j.pt)
+		etalist.append(math.fabs(j.eta))
                 dphiMET.append(j.dPhiMET)
-        return [dphiMET[j] for j in np.argsort(ptlist)[::-1]]
+	return [dphiMET[j] for j in np.lexsort((etalist, ptlist[::-1]))]
```

Comment:
Have you tested this sorting? 

At one point we talked about storing the sorted jet index. Maybe it become worthy given the issue we noticed.

Reply:
Yes I tested this sorting. 
>>> ptlist = [10, 20, 20, 30, 30, 50] 
>>> etalist = [0.1, 0.4, 0.2, 0.2, 0.3, 0.7]
>>> sort = np.lexsort((etalist, ptlist[::-1]))
>>> print sort
[5 3 4 2 1 0]

As you can see it prefers the pt sorting and only does the eta sorting when the pt is equal

Reply:
Nice